### ISSUE TITLE
[2Q2024] Parse `in_tissue` fields as categorical instead of continuous

### ISSUE THREAD

**Issue by tihuan** [chanzuckerberg/single-cell-explorer#841]
Context:
[Slack](https://chanzuckerbergteam.slack.com/archives/C06FM5ZQE3Y/p1710951748739239)

per @sidneymbell:
tl;dr -- 
`in_tissue` is a categorical, binary metadata field (`0` or `1`). Because it uses numerical values to denote true / false, it's getting parsed as continuous. It needs to be parsed as categorical so people can easily toggle between seeing all spots and only those `in_tissue`.

background -- the assay does sequencing from all the spots on the slide. Some of those spots don't actually come in contact with the tissue / are just background. Most people filter out those spots. That's why for our example datasets, we mostly see just 1 values.
In talking with some users recently, we learned that people use those spots for QC as negative controls. We're in the process of updating the data schema requirements so that data submitters have to provide all the spots, not just the `in_tissue` ones.

I'm working on finding an example dataset for us to use which has all the spots. In the meantime, we can work on making sure the metadata field is parsed correctly.

[Example](https://cellxgene.cziscience.com/e/44c9d3fd-bbde-45a5-a17c-db37a75bf697.cxg/)

DoD:
1. `in_tissue` is a categorical, binary metadata field (`0` or `1`). Because it uses numerical values to denote true / false, it's getting parsed as continuous. It needs to be parsed as categorical so people can easily toggle between seeing all spots and only those `in_tissue`.

**Comment by tihuan** [comment]
@seve @kaloster is this ticket clear enough for pointing? Thank you!

**Comment by tihuan** [comment]
Double checking with @seve to see if this is a FE or BE ticket!

CC: @kaloster

**Comment by kaloster** [comment]
@sidneymbell @brianraymor - is this something that could be treated as part of migration instead of introducing code exceptions? (maybe as part of the 5.1 validation run?)

Not sure if that's feasible but thought it's worth asking. Thanks!

cc: @dsadgat

**Comment by brianraymor** [comment]
> is this something that could be treated as part of migration instead of introducing code exceptions? (maybe as part of the 5.1 validation run?)

Hi @kaloster 

I'm not sure that I understand the question. 

**Validation** only ensures that the fields meet the `0` or `1` requirements. The small group did consider modeling as a boolean but downstream scverse/seurat frameworks model as numerics/integers.

**Comment by kaloster** [comment]
Ok got it - I appreciate the additional context, I wasn't aware of the downstream - thanks @brianraymor
